polaris/ocean/model/ocean_model_step.py

```diff
@@ -93,8 +93,9 @@ def __init__(self, component, name, subdir=None, indir=None, ntasks=None,
             max_memory=max_memory, cached=cached, yaml=yaml,
             update_pio=update_pio, make_graph=make_graph,
             mesh_filename=mesh_filename, partition_graph=partition_graph,
-            graph_filename=graph_filename)
+            graph_filename='graph.info')
 
+        self.graph_target = graph_filename
```

Comment:
Let's also rename the parameter from `graph_filename` to `graph_target` to avoid confusion.

Then, `self.graph_target` doesn't need to be an attribute since it's not used except below to add the input file.